tests/apichecks/testdata/exceptions/jsonnaming.txt

```diff
@@ -0,0 +1 @@
+[[jsonNaming] crd=cloudbuildworkerpools.cloudbuild.cnrm.cloud.google.com version=v1alpha1: field ".spec.privatePoolV1Config.networkConfig.PeeredNetworkIPRange" should use camel case]
```

Comment:
I do think it's fine to rename the v1alpha1 version

Reply:
sure.